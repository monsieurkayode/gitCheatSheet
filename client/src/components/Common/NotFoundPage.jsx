import React from 'react';
import NoContent from './NoContent';

const NotFoundPage = () => (
  <NoContent content="404. Page not found" />
);

export default NotFoundPage;
