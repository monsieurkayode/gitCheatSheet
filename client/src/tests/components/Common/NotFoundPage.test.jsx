import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../../components/Common/NotFoundPage';

describe('NotFoundPage Component', () => {
  it('should render correctly and does not crash', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
