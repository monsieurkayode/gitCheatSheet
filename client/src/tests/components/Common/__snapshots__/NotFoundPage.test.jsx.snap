// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage Component should render correctly and does not crash 1`] = `
<NoContent
  content="404. Page not found"
/>
`;
